Только в /root/dk_scripts/tmp/1: 5-Dec-2011084024LINK-W-Downfa29KLBR31-SW2-48_192.168.22.201.conf
Только в /root/dk_scripts/tmp/2: LBR31-SW2-48_192.168.22.201.conf
diff -r /root/dk_scripts/tmp/2/LBR31-SW4-48_192.168.22.203.conf /root/dk_scripts/tmp/1/LBR31-SW4-48_192.168.22.203.conf
198c198,199
< switchport mode access 
---
> switchport mode general 
> switchport general allowed vlan add 2 tagged 
Только в /root/dk_scripts/tmp/1: sw-east-1-1_192.168.200.240.conf
Только в /root/dk_scripts/tmp/1: sw-east-1-3_192.168.200.243.conf
Только в /root/dk_scripts/tmp/1: sw-east-1-4_192.168.200.244.conf
Только в /root/dk_scripts/tmp/1: sw-east-1-6_192.168.200.246.conf
Только в /root/dk_scripts/tmp/2: TCS1-SW3-48_192.168.22.75.conf
Только в /root/dk_scripts/tmp/2: TCS2-SW1-48_192.168.22.84.conf
